src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.vb

```diff
@@ -630,11 +630,7 @@ End Class]]>.Value
 
             Dim expectedOperationTree = <![CDATA[
 IInvalidStatement (OperationKind.InvalidStatement, IsInvalid) (Syntax: 'Case x')
-  Children(1):
```

Comment:
I believe I made it to return null for GetOperation for this particular case. why do you care what IsInvalidStatement return? what are you trying to get out of it?